src/sidebar/services/annotations-exporter.ts

```diff
@@ -49,22 +56,25 @@ export class AnnotationsExporter {
 
   buildTextExportContent(
     annotations: APIAnnotationData[],
-    groupName = '',
-    /* istanbul ignore next - test seam */
-    now = new Date(),
+    {
+      groupName = '',
+      displayNamesEnabled = false,
+      defaultAuthority = '',
+      /* istanbul ignore next - test seam */
+      now = new Date(),
+    }: TextExportOptions,
   ): string {
     const [firstAnnotation] = annotations;
     if (!firstAnnotation) {
       throw new Error('No annotations to export');
     }
 
+    const extractUserName = (annotation: APIAnnotationData) =>
+      annotationDisplayName(annotation, defaultAuthority, displayNamesEnabled);
+
     const { uri, title } = documentMetadata(firstAnnotation);
     const uniqueUsers = [
-      ...new Set(
-        annotations
-          .map(annotation => username(annotation.user))
-          .filter(Boolean),
-      ),
+      ...new Set(annotations.map(extractUserName).filter(Boolean)),
```

Comment:
This usage of `Array.map` is safe because the function was specifically written to be used with it. I tend to avoid using `array.map(functionNotSpecificallyWrittenForMap)` though in case the extra arguments from `map` cause breakage in future.